### PR TITLE
API network doc updates

### PR DESCRIPTION
**Pull request by SueSmith**
Updates to api network and templates sections, and one update to api docs section to bring in line with current state of product.

### CODE REVIEW

File: src/pages/docs/postman-for-publishers/api-network/add-api-network.md [hunk 1]
```diff
@@ -1,44 +1,100 @@
...
 
-For API users, you're probably wondering if there's an API out there for your specific use case. Whether it's finding sushi restaurants near you or integrating your CI/CD pipeline with your enterprise messenger, the API Network can help with that. Browse the network, and when you find what you're looking for, import the collection with a simple click of the [Run in Postman button](/docs/postman-for-publishers/run-in-postman/creating-run-button). Within moments, you'll be interacting with the API in Postman.
+Your docs can convey detailed guidance on how to make requests to your API endpoints, as well as the workflows required to integrate with your API in general. This provides the information—and actionable data—developers need to get started using your services.
 
-Can’t find what you're looking for? [Publish your own community-contributed template](/docs/postman-for-publishers/postman-templates/add-templates).
+> To add your API to the network, you will need a public team profile listing. You can [set your profile up before](#setting-up-your-profile) or during publication.
```
Comment (Raja-Simha) [inline src/pages/docs/postman-for-publishers/api-network/add-api-network.md]:
```suggestion
> To add your API to the network, you will need a public team profile listing. You can [set your profile up before](#setting-up-your-profile) or we can create one for you during publication and you can customize later. 
```
I just want to make sure we make it clear that creating or setting up a profile is not a necessary part of publishing to API Network.

File: src/pages/docs/postman-for-publishers/api-network/add-api-network.md [hunk 2]
```diff
@@ -1,44 +1,100 @@
...
+
+![API Network Detail](https://assets.postman.com/postman-docs/api-network-detail.jpg)
+
+Select the categories that best describe your API and service. API consumers can choose categories to browse APIs in the network from the left sidebar in the Postman app and at [explore.postman.com](https://explore.postman.com).
+
+![Network Categories](https://assets.postman.com/postman-docs/network-in-app.jpg)
+
+### Setting up your team profile
+
+When you publish to the API Network, your API will be listed as part of your team profile. If you don't already have a team profile, Postman will automatically generate one when you publish.
```
Comment (Raja-Simha) [inline src/pages/docs/postman-for-publishers/api-network/add-api-network.md]:
```suggestion
When you publish to the API Network, your API will be listed as part of your team profile. If you don't already have a team profile, Postman will automatically generate one when you publish and you can customize later.
```

File: src/pages/docs/postman-for-publishers/api-network/add-api-network.md [hunk 3]
```diff
@@ -1,44 +1,100 @@
...
+
+<img alt="Publish Profile" src="https://assets.postman.com/postman-docs/publish-profile.jpg" width="450px"/>
+
+Use the __Public Profile Settings__ link or navigate to your [team profile settings](https://go.postman.co/settings/team/public) to enable and fill out the details consumers will see when your API is published and visible in the network.
```
Comment (Raja-Simha) [inline src/pages/docs/postman-for-publishers/api-network/add-api-network.md]:
```suggestion
Use the __Public Profile Settings__ link or navigate to your [team profile settings](https://go.postman.co/settings/team/public) to enable or customize the details consumers will see when your API is published and visible in the network.
```

File: src/pages/docs/postman-for-publishers/api-network/add-api-network.md [hunk 4]
```diff
@@ -1,44 +1,100 @@
...
+
+![Profile Detail](https://assets.postman.com/postman-docs/profile-details.jpg)
+
+## Publish to the network
+
+Once your team profile is set up any collection docs you publish will be associated with it. Click __Publish Collection__.
+
+![Publish Team](https://assets.postman.com/postman-docs/publish-as-team.jpg)
+
+You will receive a notification and email to confirm that your collection is published.
```
Comment (Raja-Simha) [inline src/pages/docs/postman-for-publishers/api-network/add-api-network.md]: This line could be moved out of this section.

**Comment by SueSmith** [comment]
Great thanks @Raja-Simha - have added / edited suggestions and same in templates doc. :+1:

**Pull request merged**